README_BRANCHING.md

```diff
@@ -0,0 +1,41 @@
+This document attempts to capture the details of our branching strategy
+for Puppet Server.  This information is up-to-date as of 2015-03-06.
+
+## THE MOST IMPORTANT THING TO KNOW
+
+tl;dr: we are putting a freeze on merging up changes from `stable` to `master`
+until Puppet Server 2.0 ships.
```

Comment:
That sounds like a lot more work to me; we're not just talking about dependencies, we're talking about major code changes in the Puppet Server code base.

If we can tag 2.0 soon-ish, then this freeze should be short-lived and hopefully not a huge issue.  Failing that, I think the best thing to do would be to create a 2.1.x branch and start merging up to that from both 1.1 and 2.0.